frontend/src/components/case/CaseTimeline.css

```
@@ -0,0 +1,300 @@
+  border-color: var(--border-strong);
+  color: var(--text-primary);
+}
+
+/* Severity chips get a small leading dot in their own hue; the chip body stays
+   neutral so the control still reads as a filter, not an alert. */
+.case-timeline__chip[class*='--sev-']::before {
+  content: '';
+  width: 6px;
+  height: 6px;
+  border-radius: 50%;
+  background: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)